operator/controllers/lifecycle/keptnapp/controller.go

```diff
@@ -159,6 +159,9 @@ func (r *KeptnAppReconciler) createAppVersion(ctx context.Context, app *klcv1alp
 	}
 
 	appVersion := app.GenerateAppVersion(previousVersion, traceContextCarrier)
+	if len(appVersion.ObjectMeta.Name) > 253 {
```

Comment:
let's make the `253` a constant - see https://github.com/keptn/lifecycle-toolkit/blob/d576a3323ff32a28ff1a8a207d811a01e1865d08/operator/apis/lifecycle/v1alpha3/common/common.go#L38 

Also, can you please add a unit test for this case? Feel free to reach out if you have any questions about how to best implement unit tests for controllers